src/main/java/de/caritas/cob/consultingtypeservice/api/consultingtypes/ConsultingTypeMongoTenantAwareRepositoryService.java

```diff
@@ -0,0 +1,133 @@
+package de.caritas.cob.consultingtypeservice.api.consultingtypes;
+
+import de.caritas.cob.consultingtypeservice.api.exception.UnexpectedErrorException;
+import de.caritas.cob.consultingtypeservice.api.exception.httpresponses.NotFoundException;
+import de.caritas.cob.consultingtypeservice.api.model.ConsultingTypeEntity;
+import de.caritas.cob.consultingtypeservice.api.service.LogService;
+import de.caritas.cob.consultingtypeservice.api.service.tenant.TenantContext;
+import de.caritas.cob.consultingtypeservice.schemas.model.ConsultingType;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+import org.springframework.beans.BeanUtils;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Repository for {@link ConsultingType}.
+ */
+@Repository
+@Primary
+@ConditionalOnExpression("${multitenancy.enabled:true}")
+@AllArgsConstructor
+public class ConsultingTypeMongoTenantAwareRepositoryService implements
+    ConsultingTypeRepositoryService {
+
+  private static final Long TECHNICAL_TENANT_ID = 0L;
+  private @NonNull ConsultingTypeMongoTenantAwareRepository consultingTypeMongoTenantAwareRepository;
+  private @NonNull ConsultingTypeConverter consultingTypeConverter;
+
+  /**
+   * Get a complete list of all {@link ConsultingType}.
+   *
+   * @return a {@link List} of {@link ConsultingType}
+   */
+  public List<ConsultingType> getListOfConsultingTypes() {
+    if (isTechnicalTenantContext()) {
+      List<ConsultingTypeEntity> consultingTypeEntities = consultingTypeMongoTenantAwareRepository.findAll();
```

Comment:
You could again use "var" in this class for types

Reply:
done